haystack/schema.py

```diff
@@ -94,17 +96,52 @@ def __str__(self):
         return str(self.to_dict())
 
 
+@dataclass
+class Answer:
```

Comment:
Can we add doc strings as well

Reply:
Sure - once we finalized the design, I'll update docstrings and add more descriptions in our documentation